src/js/modules/articles/components/ArticleFeaturedMedia.js

```diff
@@ -57,11 +65,17 @@ class ArticleFeaturedMedia extends Component {
         imgWidth: img.width,
       });
     };
-  }
-  render() {
-    const { classes, featuredMedia, isCaption, users } = this.props;
+  };
+  getComponent = key => {
```

Comment:
I'm a little confused by this. Why is the enclosing component deciding which children to render? Why isn't a parent component deciding which children to give the enclosing component